main.nf

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl = 2
```

Comment:
This flag is no longer needed in Nextflow 22.03.0 and later

Reply:
However, on line 13 in `nextflow.config` the required Nextflow version is set to 23.04.0 and later

Reply:
Correct. Part of To-Do in this PR description refers to these lines: 

- [ ] Add ${workflow_path}/tools/Nextflow